src/index.tsx

```diff
@@ -38,7 +38,8 @@ interface PageViewEvent {
     uuid: string;             // Unique identifier for this browser/device
     is_loggedin: boolean;     // Whether the user was logged in during this visit
     event_id: string;        // Unique identifier for this event
-    deriv_user_id?: string
+    deriv_user_id?: string;
+    event_type?: 'pageview' | 'signup' | 'login'; // Optional event type
```

Comment:
let's extract it to its own type and reuse it in other placea like the `sendEventToBackend` params as well
```
interface EventType = 'pageview' | 'signup' | 'login';
```

```suggestion

    event_type?: EventType
```